x/market/spec/01_state.md

```diff
@@ -14,3 +14,4 @@ Order state consists of the following data:
 * Destination: a `Coin` representing the minimum amount of tokens to buy.
 * DestinationFilled: `Int` that tracks the bought amount so far.
 * Price: a `Dec` calculated as *Destination* / *Source*.
+* Created: the `timestamp` of the order acceptance, creation event.
```

Comment:
Perhaps mention that it's the blocktime of the block in which the order is processed.